CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-23.2.1
+
+This edge release sees the `linkerd-cni` plugin moved to
+`linkerd2-proxy-init` and released from that repository. An iptables
+improvement to `linkerd-cni` and `proxy-init` is the main focus. Other
+minor fixes are also included.
+
+* Upgrades `linkerd-cni` to v1.0.0 which is now published from
+  `linkerd2-proxy-init` and makes iptables rules idempotent (thanks @jim-minter!)
+* Upgraded `proxy-init` from v2.1.0 to v2.2.0 which makes `iptables` rules
+  idempotent (thanks @jim-minter!)
+* Add `server_port_subscribers` metrics to server and service watchers
```

Comment:
I think we can give more details here, all related to the Destination controller, perhaps as different items:
- Added the `server_port_subscribers` metric to track the number of subscribers to Server changes associated with a pod's port.
- Added the `service_subscribers` metric to track the number of subscribers to Service changes.
- Fixed a small memory leak in the opaque ports watcher.

Reply:
Perfect, that's now added.